application/src/js/jquery.datePicker.js

```diff
@@ -1168,7 +1170,7 @@
 		TEXT_NEXT_MONTH		:	'Next month',
 		TEXT_CLOSE			:	'Close',
 		TEXT_CHOOSE_DATE	:	'Choose date',
-		HEADER_FORMAT		:	'mmmm yyyy'
+		HEADER_FORMAT		:	'yyyy'
 	};
```

Comment:
This change makes January, current year, first choice?

Reply:
That is because of a weird character was next to the year and it shouldn't be there.